tests/test_remote_inventory/environment_one/inventory/targets/remoteinv-example.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
+    kapitan:
+        vars:
+            target: remoteinv-example
+        inventories:
+            - type: git
+              output_path: targets/
+              source: https://github.com/deepmind/kapitan.git
+              ref: master
+              subdir: kapitan/tests/test_remote_inventory/environment_two/inventory/targets/
```

Comment:
@uberspot @ramaro Hi! To test the recursive fetching and compiling of inventories I created two dummy directories `kapitan/tests/test_remote_inventory/environment_one` and `kapitan/tests/test_remote_inventory/environment_two`. The target files of environment_one fetches the target files of  environment_two and the target files of environment_two fetches `kapitan/tests/test_resources/inventory/`. This raises an [error on Travis](https://travis-ci.org/github/deepmind/kapitan/builds/698962601) as after cloning https://github.com/deepmind/kapitan.git, there's no directory called `kapitan/tests/test_remote_inventory/environment_two`. Is making a separate PR for the dummy environments and then testing it on Travis the only way? Or would it be better if I host the dummy environments on my github for testing and later make a PR?

Reply:
You can definitely create the two dirs in a separate PR with just the test resources.
Just add a small readme.md file in those dirs explaining what they are for if that's easy? 
So it's more intuitive. :)